media/css/firefox/channel.scss

```diff
@@ -14,10 +14,13 @@ $image-path: '/media/protocol/img';
 @import '../../protocol/css/components/emphasis-box';
 
 //* -------------------------------------------------------------------------- */
-// Global
+// Content
 
-.mzp-l-narrow {
-    max-width: $content-lg;
+#beta .mzp-l-content,
+#developer .mzp-l-content,
+#nightly .mzp-l-content,
+#testflight .mzp-l-content {
+    max-width: $content-md;
```

Comment:
This looks / feels pretty squished? `$content-lg` seems closer to what's in production.

Reply:
There wasn't a really good fit for this one. I agree lg is closer to what's in production but production already has a line length that is too wide for readability.

I would have liked to make the call outs their default width and keep the supplementary info narrower but the text inside the callout was not inline with the other info. I'll look into re-structuring the page.

Reply:
Changed it to $content-lg